### PR TITLE
apply custom CSS to admin panel

### PR DESCRIPTION
**Pull request by type-face**
## What does this PR do?
Applies styles from custom CSS file to the admin panel.
See https://github.com/coralproject/talk/issues/1780

## How do I test this PR?
1. View admin panel
2. Create .css file that overrides styles on an existing element on the admin panel
3. Update custom admin CSS url to reference .css file from step 2
4. Refresh admin panel to see updated styles

### CODE REVIEW

**Comment by wyattjoh** [comment]
We actually initially had this feature, but decided against it as there were some conflicting rules that applied to the stream as well as the admin feature. I would be able to accept this if it included another separate custom CSS file for admin versus the stream.

**Comment by type-face** [comment]
Yes, I saw that this existed before but wasn't sure of the reason for its removal. I'll revise to use a separate CSS file for the admin panel.

File: locales/en.yml [hunk 1]
```diff
@@ -124,6 +124,8 @@ en:
     copy_and_paste: 'Copy and paste code below into your CMS to embed your comment box in your articles'
     custom_css_url: 'Custom CSS URL'
     custom_css_url_desc: 'URL of a CSS stylesheet that will override default Embed Stream styles. Can be internal or external.'
+    custom_admin_css_url: 'Custom Admin Panel CSS URL'
```
Comment (type-face) [inline locales/en.yml:127]: I'm sure how to indicate that other translations are needed.

Comment (wyattjoh) [reply locales/en.yml:127]: If a translation doesn't exist in the target language, English is used as the fallback. Typically, language authors just notice that change and resolve it, we're planning on some tooling in the future to help with this, but we have limited resources unfortunately.

**Comment by type-face** [comment]
@kgardnr I noticed that the milestone for this was removed. I'm assuming that this is still just awaiting review, but if I need to take any other action to keep this moving, please let me know. Thanks!

**Comment by wyattjoh** [comment]
That's correct @type-face, we had to release earlier to resolve a breaking bug.

When we have some cycles to review this PR it can be included in the next release.

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**